.github/workflows/django-sample-app-ecr-deploy.yml

```diff
@@ -53,10 +53,10 @@ jobs:
 
       - name: Upload Main Service Image
         run: |
-          docker tag pythonsampleapp/frontend-service ${{ env.ACCOUNT_ID }}.dkr.ecr.${{ matrix.aws-region }}.amazonaws.com/${{ secrets.APP_SIGNALS_PYTHON_E2E_FE_SA_IMG }}:latest
-          docker push ${{ env.ACCOUNT_ID }}.dkr.ecr.${{ matrix.aws-region }}.amazonaws.com/${{ secrets.APP_SIGNALS_PYTHON_E2E_FE_SA_IMG }}:latest
+          docker tag pythonsampleapp/frontend-service ${{ env.ACCOUNT_ID }}.dkr.ecr.${{ matrix.aws-region }}.amazonaws.com/${{ secrets.APP_SIGNALS_PYTHON_E2E_FE_SA_IMG }}
```

Comment:
What tag will be if we remove the latest tag? And will the latest image be used in terraform?

Reply:
The `latest` tag seems to be duplicated (already has `latest` in the GitHub secret itself). So after this change, it should still use `latest`.

In a situation where there is no tag, `latest` should be the default tag still.